indexer/modules/custom/uniswap_v3/uniswap_v3_pool_job.py

```diff
@@ -1,38 +1,75 @@
-import configparser
 import json
 import logging
-import os
-from dataclasses import fields
-from itertools import groupby
-from operator import attrgetter
+from collections import defaultdict
 
-from common.utils.format_utils import bytes_to_hex_str, hex_str_to_bytes
+import eth_abi
+
+from common.utils.abi_code_utils import decode_log
+from indexer.domain import dict_to_dataclass
 from indexer.domain.log import Log
-from indexer.domain.transaction import Transaction
 from indexer.executors.batch_work_executor import BatchWorkExecutor
 from indexer.jobs import FilterTransactionDataJob
 from indexer.modules.custom import common_utils
-from indexer.modules.custom.feature_type import FeatureType
-from indexer.modules.custom.uniswap_v3 import constants, util
-from indexer.modules.custom.uniswap_v3.constants import UNISWAP_V3_ABI
-from indexer.modules.custom.uniswap_v3.domain.feature_uniswap_v3 import (
+from indexer.modules.custom.uniswap_v3.domains.feature_uniswap_v3 import (
     UniswapV3Pool,
     UniswapV3PoolCurrentPrice,
     UniswapV3PoolPrice,
-    UniswapV3SwapEvent,
```

Comment:
Why do you delete this?

Reply:
Because the base of this is pulled from the fbtc branch, but that branch doesn't have this domain. But I can add it back.